src/components/text-field/style/lumapps/_index.scss

```diff
@@ -60,7 +60,7 @@
     }
 
     &__input-native {
-        flex: 1;
+        width: 100%;
```

Comment:
Why?

Reply:
`flex: 1` does not work well with resizing the text area. In browsers that do not support the `resize` css property, it allows changes like this:
![Screenshot 2019-10-22 at 10 13 41](https://user-images.githubusercontent.com/13719066/67268413-b5a08f80-f4b4-11e9-8519-818fa3e55c06.png)